Sources/ZcashLightClientKit/Rust/ZcashRustBackendWelding.swift

```diff
@@ -77,182 +244,96 @@ protocol ZcashRustBackendWelding {
-    Checks that the scanned blocks in the data database, when combined with the recent
-    `CompactBlock`s in the cache database, form a valid chain.
-    This function is built on the core assumption that the information provided in the
-    cache database is more likely to be accurate than the previously-scanned information.
-    This follows from the design (and trust) assumption that the `lightwalletd` server
-    provides accurate block information as of the time it was requested.
-        - Returns:
-            * `-1` if the combined chain is valid.
-            * `upper_bound` if the combined chain is invalid.
-            * `upper_bound` is the height of the highest invalid block (on the assumption that the highest block in the cache database is correct).
-            * `0` if there was an error during validation unrelated to chain validity.
-    - Important: This function does not mutate either of the databases.
-    */
+    /// Checks that the scanned blocks in the data database, when combined with the recent
+    /// `CompactBlock`s in the cache database, form a valid chain.
+    /// This function is built on the core assumption that the information provided in the
+    /// cache database is more likely to be accurate than the previously-scanned information.
+    /// This follows from the design (and trust) assumption that the `lightwalletd` server
+    /// provides accurate block information as of the time it was requested.
+    ///    - Returns:
+    ///        * `-1` if the combined chain is valid.
+    ///        * `upper_bound` if the combined chain is invalid.
+    ///        * `upper_bound` is the height of the highest invalid block (on the assumption that the highest block in the cache database is correct).
+    ///        * `0` if there was an error during validation unrelated to chain validity.
+    /// - Important: This function does not mutate either of the databases.
     static func validateCombinedChain(dbCache: URL, dbData: URL, networkType: NetworkType) -> Int32
-    
-    /**
-    Returns the nearest height where a rewind is possible. Currently prunning gets rid of sapling witnesses older
-    than 100 blocks. So in order to reconstruct the witness tree that allows to spend notes from the given wallet
-    the rewind can't be more than 100 block or back to the oldest unspent note that this wallet contains.
-    - Parameters:
-        - dbData: location of the data db file
-        - height: height you would like to rewind to.
-    */
-    static func getNearestRewindHeight(dbData: URL, height: Int32, networkType: NetworkType) -> Int32
 
-    /**
-    rewinds the compact block storage to the given height. clears up all derived data as well
-    - Parameters:
-        - dbData: location of the data db file
-        - height: height to rewind to. DON'T PASS ARBITRARY HEIGHT. Use getNearestRewindHeight when unsure
-    */
+    /// rewinds the compact block storage to the given height. clears up all derived data as well
```

Comment:
This documentation is a little confusing, because I think of the "compact block storage" as the cache db. 
```suggestion
    /// Resets the state of the database to only contain block and transaction information up to the 
    /// specified height.
```

Reply:
I think that this should also document what happens if the user has sent transactions in blocks that would be invalidated by this rewind.